web/pnpm-lock.yaml

```diff
@@ -0,0 +1,3996 @@
+lockfileVersion: '6.0'
```

Comment:
The choice of package manager in a front-end development environment is typically at the discretion of the developer.
Should a developer employ `Yarn`, the `yarn.lock` file becomes instrumental.
However, I surmise that it may be deemed superfluous in future considerations and thus, could be discarded.

Reply:
I remove `yarn.lock`.